tests/e2e/specs/stories-editor/global-inserter.test.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * WordPress dependencies
+ */
+import { createNewPost, getAllBlocks } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { activateExperience, deactivateExperience, insertBlock } from '../../utils';
+
+const INSERTER_SELECTOR = '#amp-story-inserter';
+const CTA_BLOCK_NAME = 'Call to Action';
+
+/**
+ * Tests that a given block is present in the inserter.
+ *
+ * @param {string} block The name of the block, like 'Image'.
+ */
+const testBlockPresentInInserter = async ( block ) => {
+	// It should be possible to insert the block.
+	await insertBlock( block );
+
+	await page.click( INSERTER_SELECTOR );
+
+	// The block should be present in the inserter.
+	const inserter = await page.$( INSERTER_SELECTOR );
+	await expect( inserter ).toMatch( block );
+	await page.click( INSERTER_SELECTOR );
+};
+
+describe( 'Global Inserter', () => {
+	beforeAll( async () => {
+		await activateExperience( 'stories' );
+	} );
+
+	afterAll( async () => {
+		await deactivateExperience( 'stories' );
+	} );
+
+	it.each( [
+		'Page',
+		'Image',
+		'Text',
+		'Video',
+		'Quote',
+		'Story Author',
+		'List',
+		'Story Title',
+		'Story Date',
+		'Custom HTML',
+		'Code',
+		'Preformatted',
+		'Pullquote',
+		'Table',
+		'Verse',
+		'Embed',
+	] )( 'contains the block %s',
+		async ( block ) => {
+			await createNewPost( { postType: 'amp_story' } );
+			await testBlockPresentInInserter( block );
+
+			// It should still be possible to select the block, even with an Image innerBlock inserted and selected.
+			await insertBlock( 'Image' );
+			await testBlockPresentInInserter( block );
+		}
+	);
+
+	it( 'should only show the CTA block when not on the first page', async () => {
```

Comment:
This is already covered in the CTA tests.

Reply:
Or well, it is testing the presence in the Inserter, so I guess it's a bit different.